src/Microsoft.AspNetCore.OData/Extensions/ODataServiceCollectionExtensions.cs

```diff
@@ -139,5 +139,36 @@ public static IServiceCollection AddODataQueryFilter(this IServiceCollection ser
             services.TryAddEnumerable(ServiceDescriptor.Singleton<IFilterProvider>(new QueryFilterProvider(queryFilter)));
             return services;
         }
+
+        /// <summary>
+        /// Enables support for case insensitive model binding.
+        /// </summary>
+        /// <param name="services">The services collection.</param>
+        public static IServiceCollection EnableODataCaseInsensitiveModelBinding(this IServiceCollection services)
+        {
+            if (services == null)
```

Comment:
You could remove this as it will be checked again for null in the other call.

Reply:
#Resolved